Packs/CrowdStrikeFalcon/Classifiers/classifier-CrowdStrike_Falcon_Mapper.json

```diff
@@ -1,101 +1,276 @@
+                    "simple": "device.service_provider"
+                },
+                "Description": {
+                    "simple": "behaviors.description"
+                },
+                "Detected User": {
+                    "simple": "behaviors.user_name"
+                },
+                "Device External IP": {
+                    "simple": "device.external_ip"
+                },
+                "Device Local IP": {
+                    "simple": "device.local_ip"
+                },
+                "Device Model": {
+                    "simple": "device.platform_name"
+                },
+                "Device Name": {
+                    "simple": "device.hostname"
+                },
+                "Agent Version": {
+					"simple": "device.agent_version"
 				},
-				"MD5": {
+                "File MD5": {
+                    "simple": "behaviors.md5"
+                },
+                "MD5": {
 					"complex": null,
 					"simple": "behaviors.md5"
 				},
-				"OS": {
-					"complex": null,
-					"simple": "device.os_version"
-				},
-				"SHA256": {
+                "SHA256": {
 					"complex": null,
 					"simple": "behaviors.sha256"
-				}
-			}
-		},
-		"CrowdStrike Falcon Incident": {
-			"dontMapEventToLabels": false,
-			"internalMapping": {
-				"Device External IP": {
-					"complex": null,
-					"simple": "hosts.external_ip"
 				},
-				"Device Local IP": {
-					"complex": null,
-					"simple": "hosts.local_ip"
-				},
-				"Device Model": {
-					"complex": null,
-					"simple": "hosts.platform_name"
-				},
-				"Device Name": {
-					"complex": null,
-					"simple": "hosts.hostname"
-				},
-				"MAC Address": {
-					"complex": null,
-					"simple": "hosts.mac_address"
+                "File Name": {
+                    "simple": "behaviors.filename"
+                },
+                "File Paths": {
+                    "simple": "behaviors.filepath"
+                },
+                "File SHA256": {
+                    "simple": "behaviors.sha256"
+                },
+                "Hostnames": {
+                    "simple": "device.hostname"
+                },
+                "Last Update Time": {
+                    "simple": "last_behavior"
+                },
+                "MAC Address": {
+                    "simple": "device.mac_address"
+                },
+                "OS": {
+                    "simple": "device.os_version"
+                },
+                "Parent CMD line": {
+                    "simple": "behaviors.parent_details.parent_cmdline"
+                },
+                "Start Time": {
+                    "simple": "first_behavior"
+                },
+                "State": {
+                    "simple": "status"
+                },
+                "name": {
+                    "complex": {
+                        "accessor": "display_name",
+                        "filters": [],
+                        "root": "behaviors.[0]",
```

Comment:
Is this the first object in the array on purpose?

Reply:
As here we are talking about the detection alert - the whole information about the alert is under "_behaviors_" key. Therefore, providing a human-readable name for the Xsoar incident, I decided to enrich it only with the first behavior detection details (Of course avoiding a situation where we will flood the name with multiple and probably duplicated directions display names). Here is an example:

_Falcon Detection - SampleTemplateDetection - Detection ID: ldt:15dbb9d8f06b45fe9f61eb46e829d986:420692447105_


Instead of:
_Detection ID: ldt:15dbb9d8f06b45fe9f61eb46e829d986:420698775497_